python/src/scipp/_unary.py

```diff
@@ -45,7 +45,7 @@ def isneginf(x: _cpp.Variable) -> _cpp.Variable:
 def to_unit(x: _cpp.Variable,
             unit: _Union[_cpp.Unit, str],
             *,
-            copy: bool = False) -> _cpp.Variable:
+            copy: bool = True) -> _cpp.Variable:
```

Comment:
The new defaults should be mentioned in the release notes in the "Breaking changes" section.